### ISSUE TITLE
🐛 git commit fails with "nonzero exit code"

### ISSUE THREAD

**Issue by davidalpert** [davidalpert/go-git-mob#68]
**Describe the bug**
`prepare-commit-msg` hook script blocks committing

**To Reproduce**
1. Download and install `go-git-mob`
2. Change directory into a local git repo
3. Initialize the `prepare-commit-msg` hook script`
```
git mob init
```
4. Start soloing
```
git mob solo
```
5. Try to commit
```
git commit --allow-empty -m "test commit"
```

**Expected behaviour**
The commit succeeds with no `Co-authored-by` annotation (because you are soloing with `git solo`).

**Actual behaviour**
The commit fails with an error message like this in `v0.7.0`:
```
reading co-authors: nonzero exit code: 1: 
```
And like this in `v0.7.1`:
```
GetAllGlobal(git-mob.co-author): nonzero exit code: 1:
```

**Desktop (please complete the following information):**
 - OS: MacOS
 - Version: seen in
     - v0.7.0
     - v0.7.1

**Workaround**
- delete the `prepare-commit-msg` hook script
```
rm -f .git/hooks/prepare-commit-msg
```
this will disable automatic injection of the `Co-authored-by` annotation but will allow commits to succeed.

**Comment by davidalpert** [comment]
same root cause as #67
```
git config --global --get-all git-mob.co-author
```
returns a nonzero exit code when the `git-mob.co-author` key is missing

**Comment by davidalpert** [comment]
resolved with `v0.7.2`
